src/builder/create_interaction.rs

```diff
@@ -169,3 +179,44 @@ impl CreateInteraction {
         self
     }
 }
+
+#[derive(Clone, Debug, Default)]
+pub struct CreateInteractions(pub Vec<Value>);
+
+impl CreateInteractions {
+    /// Creates a new interaction
```

Comment:
There should be a full stop.
```suggestion
    /// Creates a new interaction.
```